scripts/ogr-convert.sh

```diff
@@ -94,7 +94,17 @@ if [ -n "$source_srs" ]; then
     # no target srs, but if s_srs is provided, t_srs must be as well
     target_srs=$source_srs
   fi
+else
+  # Extract driverShortName
+  driverShortName=$(ogrinfo -json $source_loc | jq -r '.driverShortName')
+  echo "Driver Short Name: $driverShortName"
+  # Check if driverShortName is GML and source_srs is not set
+  if [ "$driverShortName" == "GML" ] && [ -z "$source_srs" ]; then
```

Comment:
`&& [ -z "$source_srs" ]` seems redundant because we are already in the else case.